README.md

```diff
@@ -801,19 +800,19 @@ $ ballerina run --config <path-to-conf>/ballerina.conf students
 #### Views of traces
  After making some HTTP requests, go to the distributed message tracing dashboard in your WSO2 Stream Processor portal.
 
- - You are expected to see the traces as below when you press the search button in the dashboard.
+ - You are expected to see the traces as indicated below when you press the search button in the dashboard.
  
 ![SP](images/trace1.png "SP")
  
- - To view a particular trace click on the trace row. And you will see as below
+ - To view a particular trace, click on the trace row. You will see the following.
  
 ![SP](images/trace2.png "SP")
     
- - To view span details with metrics click on a particular span and you are expected to see as below
+ - To view span details with metrics, click on a particular span and you are expected to see the following.
```

Comment:
To view span details with the metrics as shown below, click on a particular span.